### DISCUSSION TITLE
May I ask, is there any other solution that can replace its default serialization tool with Gson?

### DISCUSSION THREAD

**Discussion by ghost** [logfellow/logstash-logback-encoder#984] [Q&A] [answered]
I really want to use the `logstash-logback-encoder` in my project, but my project does not use Jackson. May I ask if there is any other solution to replace it with Gson?

My development  environment is :point_down:
| Component                | version |
| ------------------------ | ------- |
| Spring Boot              | 2.7.10  |
| logback                  | 1.2.11  |
| logstash-logback-encoder | 7.3     |

**Comment by philsttr** [comment, marked as answer]
logstash-logback-encoder is built around jackson, uses its features extensively, and exposes jackson interfaces as part of logstash-logback-encoder's public interface.  It is not feasible for logstash-logback-encoder to use gson.

**Comment by ghost** [comment]
Thanks for reply~
